2200014/problem4.py

```diff
@@ -0,0 +1,9 @@
+a=int(input())
+for i in range(a):
+    T=int(input())
+    income=list(map(int,input().split()))
+    cnt=0
+    for j in income:
+        if j<=(sum(income)/len(income)):
+            cnt+=1
```

Comment:
```suggestion
    avg = sum(income)/len(income)
    for j in income:
        if j<=avg:
            cnt+=1
```
평균을 `for` 문 밖에서 한 번만 구하는 방법도 있습니다!